src/lib/preferences/preferences.cpp

```diff
@@ -214,7 +214,13 @@ Preferences::Preferences(BrowserWindow* window)
     //APPEREANCE
     settings.beginGroup("Browser-View-Settings");
     ui->showStatusbar->setChecked(settings.value("showStatusBar", false).toBool());
+    // NOTE: instantBookmarksToolbar and showBookmarksToolbar cannot be both enabled at the same time
```

Comment:
@Martii why not just make it drop down combobox with 3 options (display, display on focus, do not display) ?